avogadro/quantumio/nwchemlog.cpp

```diff
@@ -56,12 +56,16 @@ std::vector<std::string> NWChemLog::mimeTypes() const
   return std::vector<std::string>();
 }
 
-bool NWChemLog::read(std::istream& in, Core::Molecule& molecule)
+[[nodiscard]] bool NWChemLog::read(std::istream& in, Core::Molecule& molecule)
```

Comment:
For now, we're sticking to C++ 11, thanks.

Reply:
Contributing.MD says C++17 is OK to use, but alright.

Reply:
Nodiscard removed.